code/modules/research/protolathe.dm

```diff
@@ -60,10 +60,10 @@ Note: Must be placed west/left of and R&D console to function.
 	for(var/obj/item/weapon/stock_parts/matter_bin/M in component_parts)
 		T += M.rating
 	materials.max_amount = T * 75000
-	T = 0
+	T = 1.2
 	for(var/obj/item/weapon/stock_parts/manipulator/M in component_parts)
-		T += (M.rating/3)
-	efficiency_coeff = max(T, 1)
+		T -= M.rating/10
+	efficiency_coeff = min(max(0, T), 1)

```

Comment:
It used to make the coeff to be 1 or higher. I have no idea why it was divided by 3, that just caused a bug in which changing one manipulator to nano manipulator did nothing. The new part starts with T=1.2 and makes it smaller for every improved component rating, meaning 1.2-0.2=1 with normal parts and down to 1.2-0.8=0.4 with T4 parts. The minmax check is there just to prevent any admin-modified changes to cause coeff to go to weird values (above the max cost or below free cost).